cli/flow-typed/npm/yargs_v4.x.x.js

```diff
@@ -142,6 +142,8 @@ declare module "yargs" {
 
     pkgConf(key: string, cwd?: string): this;
 
+    recommendCommands(): this;
```

Comment:
The CLI [uses `yargs` version `^12.0.2`](https://github.com/flow-typed/flow-typed/blob/0b0eb4afe995b298c2b827beaaf5dc5b5982bd0d/cli/package.json#L43) So it would be good to update the lib-def here, since `recommendCommands` is part of the `10.x.x` def.